monkestation/code/modules/projectiles/projectile/special/rocket.dm

```diff
@@ -34,4 +34,30 @@
 		new/obj/spatial_rift(A.loc)
 		qdel(A)
 
+	if(istype(A, /obj/anomaly/energy_ball))
```

Comment:
Can't this get combined into the singularity one?
If /obj/anomaly is just singularities and energy balls, that immediately applies to all subtypes as well.